src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.jsx

```diff
@@ -37,13 +39,18 @@ export default class InstalledIntegration extends React.Component {
     );
   }
 
+  mergeIntegration() {}
+
   renderDisableIntegration(integration) {
-    const message = `You must uninstall this integration from
-      %s in order to delete this integration in Sentry.`;
+    let {body, action_text} = integration.provider.aspects.disable_dialog;
```

Comment:
nit: I've been trying to use `const` for new components that aren't using `let`. Not trying to force it on you, but I think consistency within files at least is good.